### PR TITLE
added host.json change

### LINKED ISSUE #540: Add host.json entry for API calls
References user story #434 

The addition to the host.json file was not included in the deploy fix PR

### PR DESCRIPTION
**Pull request by thessbsu**
Closes #540 

Added host.json change. Tested working without the "-service" on the end of the api calls over the network

### CODE REVIEW

**Comment by thessbsu** [comment]
not sure why tests are failing

**Comment by thessbsu** [comment]
added proxy items and commented the creation in docker-compose. Future teams may need to test deploy

File: BEIMA.Backend/host.json [hunk 1]
```diff
@@ -7,5 +7,10 @@
                 "excludedTypes": "Request"
             }
         }
+    },
+    "extensions": {
+        "http": {
+          "routePrefix": "beima"
```
Comment (AshlynAdamson) [inline BEIMA.Backend/host.json:13]: Could the functional tests be failing because of this added prefix?

Comment (josephmoore99) [reply BEIMA.Backend/host.json:13]: Yeah, I think you need to add the correct url prefix in the FT BeimaClient.cs, and the FT base, I can branch from this and do that real quick if needed?

Comment (thessbsu) [reply BEIMA.Backend/host.json:13]: You'll likely get it done faster than I will

**Review by Kenny-Miller — COMMENTED** [review]
I think we will want a proxy added for the backed as well
to tell it requests on /beima-service/ should be directed to the backend

**Comment by thessbsu** [comment]
> I think we will want a proxy added for the backed as well to tell it requests on /beima-service/ should be directed to the backend

while testing I removed the "-service" part and it worked with just the redirect on all "beima" paths

**Review by Kenny-Miller — APPROVED** [review]

**Review by josephmoore99 — APPROVED** [review]

**Review by AshlynAdamson — APPROVED** [review]

**Pull request merged**